packages/serverless/test/integrations/aws_lambda.test.ts

```diff
@@ -0,0 +1,753 @@
+const AWS = require('aws-sdk');
+const fs = require('fs');
+const childProcess = require('child_process');
+const uuid = require('uuid');
+const lambda = new AWS.Lambda({ apiVersion: '2015-03-31', region: 'us-east-2' });
+
+function runLambdaFunction(additionalLambdaPrelude: any, callback: any) {
```

Comment:
Unit tests shouldn't rely on real AWS. It should be able to pass them all even without internet access. Everything else should be mocked. cc @shantanu73

Reply:
@kamilogorek makes sense actually. 
I think the original test framework for AWS python (developed before we started working on it using boto3 library) was more of automation framework than unit test framework which had dependency on internet and actual AWS environment. We developed all other frameworks based on that. 

But, even I think that this should be unit test framework and should not depend on actual environment for GCP Python integration as well.
cc @untitaker @rhcarvalho

Reply:
If you are using private AWS APIs you will have to do some sort of non-unit test or else you're relying on users to tell you when AWS changed something in their infrastructure that breaks your instrumentation.

Using `require.main` seems like something that has to be asserted to actually work from time to time.